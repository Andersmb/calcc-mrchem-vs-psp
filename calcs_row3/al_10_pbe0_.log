Starting job 5688416 on c1-[26,39],c2-32,c5-[34-36] at Sat May 21 08:46:19 CEST 2022

Copying al_10_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/triplets
Copying al_10_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/triplets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688416      al_10_pbe+        180                                             00:30:23      0:0 
5688416.bat+      batch         36        1   00:00:02          0   00:00:02   00:30:23      0:0 
5688416.ext+     extern        180        6   00:00:00          0   00:00:00   00:30:23      0:0 
5688416.0      mrchem.x        180       30   00:29:58         29   01:11:38   00:30:05      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688416                                                                          
5688416.bat+     19328K          0     19328K        0              0          0 
5688416.ext+          0          0          0        0              0          0 
5688416.0      7817240K         29 3605431091        0             23          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688416                                                                                               
5688416.bat+       16.91M               0         16.91M        0.03M                0          0.03M 
5688416.ext+        0.00M               0          0.00M            0                0              0 
5688416.0           6.77M               7          6.73M       67.01M               16         37.87M 

Job 5688416 completed at Sat May 21 09:16:42 CEST 2022
