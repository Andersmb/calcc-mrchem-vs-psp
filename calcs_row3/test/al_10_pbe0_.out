                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -1.153599
    1       N            :        0.000000        0.000000        2.222207
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              11
 Beta electrons          :                                               9
 Total electrons         :                                              20
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999998019869e-01
    1    1    a          :                              9.999999938556e-01
    2    1    a          :                              9.999999175748e-01
    3    1    a          :                              9.999998945055e-01
    4    1    a          :                              9.999999867885e-01
    5    1    a          :                              9.999999532309e-01
    6    1    a          :                              1.000000030874e+00
    7    1    a          :                              9.999989584909e-01
    8    1    a          :                              1.000002283187e+00
    9    1    a          :                              1.000002258682e+00
   10    1    a          :                              9.999994885172e-01
   11    1    b          :                              1.000000030679e+00
   12    1    b          :                              9.999999915839e-01
   13    1    b          :                              9.999999171508e-01
   14    1    b          :                              9.999998944945e-01
   15    1    b          :                              9.999999867885e-01
   16    1    b          :                              9.999999532614e-01
   17    1    b          :                              1.000000030819e+00
   18    1    b          :                              9.999989585670e-01
   19    1    b          :                              1.000002283753e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                294.530937388636
 E-N energy              :            (au)               -758.506693872469
 Coulomb energy          :            (au)                167.375109380623
 Exchange energy         :            (au)                 -6.207128389680
 X-C energy              :            (au)                -19.217641811695
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 26.956521272897
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -322.025417304585
 Nuclear energy          :            (au)                 26.956521272897
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.950688960317e+02
                         :      (kcal/mol)             -1.851585262673e+05
                         :        (kJ/mol)             -7.747032744050e+05
                         :            (eV)             -8.029233346197e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -55.926695011929
    1    1    a          :            (au)                -13.815367319923
    2    1    a          :            (au)                 -4.287816501590
    3    1    a          :            (au)                 -2.815700386128
    4    1    a          :            (au)                 -2.813715790530
    5    1    a          :            (au)                 -2.809619496558
    6    1    a          :            (au)                 -0.538916174649
    7    1    a          :            (au)                 -0.351378718095
    8    1    a          :            (au)                 -0.070506962187
    9    1    a          :            (au)                 -0.032492765830
   10    1    a          :            (au)                 -0.015420148228
   11    1    b          :            (au)                -55.923323709960
   12    1    b          :            (au)                -13.794968870907
   13    1    b          :            (au)                 -4.281133249018
   14    1    b          :            (au)                 -2.810351816153
   15    1    b          :            (au)                 -2.809643641905
   16    1    b          :            (au)                 -2.802300626093
   17    1    b          :            (au)                 -0.454970355545
   18    1    b          :            (au)                 -0.078794439399
   19    1    b          :            (au)                  0.010124441354
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -166.422991543271
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.472136e+00        -295.068896031688         -2.950689e+02
    1          6.163028e-01        -296.714222808349         -1.645327e+00
    2          3.764351e-01        -296.860126118770         -1.459033e-01
    3          1.847595e-01        -296.878570073078         -1.844395e-02
    4          8.461474e-02        -296.888354527036         -9.784454e-03
    5          8.374118e-02        -296.892869864582         -4.515338e-03
    6          5.454106e-02        -296.894699669571         -1.829805e-03
    7          1.005415e-01        -296.897791278784         -3.091609e-03
    8          7.828135e-02        -296.899083974603         -1.292696e-03
    9          3.587801e-02        -296.899755754141         -6.717795e-04
   10          2.700481e-02        -296.900015831982         -2.600778e-04
   11          1.636319e-02        -296.900122443167         -1.066112e-04
   12          9.580755e-03        -296.900159027983         -3.658482e-05
   13          4.394946e-03        -296.900170356945         -1.132896e-05
   14          2.373884e-03        -296.900175038291         -4.681345e-06
   15          1.251388e-03        -296.900176552647         -1.514356e-06
   16          8.702291e-04        -296.900176982787         -4.301401e-07
   17          4.940952e-04        -296.900177181053         -1.982664e-07
   18          3.389735e-04        -296.900177238668         -5.761461e-08
   19          2.539153e-04        -296.900177267732         -2.906472e-08
   20          2.563971e-04        -296.900177295779         -2.804700e-08
   21          1.989471e-04        -296.900177310321         -1.454180e-08
   22          1.448903e-04        -296.900177320651         -1.032942e-08
   23          1.170983e-04        -296.900177326159         -5.508412e-09
   24          7.464948e-05        -296.900177329735         -3.576190e-09
   25          7.145623e-05        -296.900177332413         -2.678007e-09
---------------------------------------------------------------------------
                      SCF converged in 25 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -1.153599
    1       N            :        0.000000        0.000000        2.222207
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                296.320062315041
 E-N energy              :            (au)               -761.217902599258
 Coulomb energy          :            (au)                166.297875419531
 Exchange energy         :            (au)                 -6.160081474769
 X-C energy              :            (au)                -19.096652265857
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 26.956521272897
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -323.856698605311
 Nuclear energy          :            (au)                 26.956521272897
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.969001773324e+02
                         :      (kcal/mol)             -1.863076726239e+05
                         :        (kJ/mol)             -7.795113027642e+05
                         :            (eV)             -8.079065046806e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -56.180085545532
    1    1    a          :            (au)                -14.414659853538
    2    1    a          :            (au)                 -4.206916514799
    3    1    a          :            (au)                 -2.737683471618
    4    1    a          :            (au)                 -2.735596061716
    5    1    a          :            (au)                 -2.734749504547
    6    1    a          :            (au)                 -0.790291780604
    7    1    a          :            (au)                 -0.385281229373
    8    1    a          :            (au)                 -0.331972916131
    9    1    a          :            (au)                 -0.278408717188
   10    1    a          :            (au)                 -0.266479915711
   11    1    b          :            (au)                -56.178757908448
   12    1    b          :            (au)                -14.380851215875
   13    1    b          :            (au)                 -4.205325836840
   14    1    b          :            (au)                 -2.735939591163
   15    1    b          :            (au)                 -2.734840913330
   16    1    b          :            (au)                 -2.729675003875
   17    1    b          :            (au)                 -0.675655924927
   18    1    b          :            (au)                 -0.298265781405
   19    1    b          :            (au)                 -0.242632170521
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -169.244069857144
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000092       -0.000092       -1.645099
 Magnitude               :            (au)                        1.645099
                         :         (Debye)                        4.181424
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.558662
 Magnitude               :            (au)                        0.558662
                         :         (Debye)                        1.419978
---------------------------------------------------------------------------
 Total vector            :       -0.000092       -0.000092       -1.086436
 Magnitude               :            (au)                        1.086436
                         :         (Debye)                        2.761445
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 29m 59s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
